provision/ansible/playbooks/tasks/start-sync-gateway.yml

```diff
@@ -1,6 +1,3 @@
-# Add sync_gateway user, copy new config
-- name: add sync gateway user
-  user: name=sync_gateway createhome=yes
 - name: make service install script executable
```

Comment:
Could we also pull these steps out into the install-source and install-package script as well?